examples/python/tools/exa-tool/praison_ai_real_estate_chatbot.py

```
@@ -0,0 +1,65 @@
+
+# """## 2. Set Your OpenAI API Key"""
+
+import os
+
+# Enter your OpenAI API key here
+OPENAI_API_KEY = "Enter your api key here"  # <-- Replace with your key
+os.environ["OPENAI_API_KEY"] = OPENAI_API_KEY
+
+# """## 3. Create the Praison AI Agent"""
+
+from praisonaiagents import Agent
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)